syntax = "proto2";

package xiso;

message ComponentParam {
    required string module_name = 1;
    required bool enable_module = 2;
}

message ApplicationParam {
    repeated ComponentParam components = 1;
}